runtime/lua/vim/treesitter.lua

```diff
@@ -506,6 +506,26 @@ function M.inspect_tree(opts)
   require('vim.treesitter.playground').inspect_tree(opts)
 end
 
+--- Determines diagnostics for a buffer containing tree-sitter queries (|treesitter-query|).
```

Comment:
```suggestion
--- Determines diagnostics for a buffer containing treesitter queries (|treesitter-query|).
```

("official" spelling in Neovim; we should try to be consistent going forward)